src/jolie/router/router.ol

```diff
@@ -1,7 +1,5 @@
-
 /*
 The MIT License (MIT)
-Copyright (c) 2016 Fabrizio Montesi <famontesi@gmail.com>
```

Comment:
it is because now the router.ol is just the starter, the body code of the router is inside the file __router.ol where there are both me and you.